### PR TITLE
In 150138171 tab not select

### PR DESCRIPTION
**Pull request by ThomasOSI**
### Pivotal Story

- [Multi-select field - tabbing selects the option](https://www.pivotaltracker.com/story/show/150138171)

### Purpose
Change functionality so multi-select fields do not select when the user tabs out

### Background
Default for react-select component is to select when the user tabs out

### Notes for Reviewer
Feature test was missing for worker safety card

### CODE REVIEW

File: app/javascript/screenings/WorkerSafetyCardView.jsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ export default class WorkerSafetyCardView extends React.Component {
       safetyInformation: this.props.screening.get('safety_information'),
     }
     return (
-      <div className='card show double-gap-top' id='worker-safety-card'>
+      <div className={`card ${mode} double-gap-top`} id='worker-safety-card'>
```
Comment (jgedeon-osi) [inline app/javascript/screenings/WorkerSafetyCardView.jsx]: 👍

File: spec/features/screening/worker_safety_spec.rb [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'spec_helper'
+require 'support/factory_girl'
+
+feature 'worker safety card' do
```
Comment (jgedeon-osi) [inline spec/features/screening/worker_safety_spec.rb]: Do we need a test for adding worker saftey stuff if there are none on the screening or it is a new screening?
